README.md

```diff
@@ -35,6 +35,30 @@ const responseBody = await sendPost(client, {
 })
 ```
 
```

Comment:
@CarlosGamero for both cases it would be useful to explain what kind of a response is returned for cases when there is an empty or non-json response that is considered valid?

Reply:
Good point! added a small note with the expected outcome, please have a look to see if it is good enough 🙏